groovy/prePostInit/oreDict.groovy

```diff
@@ -32,14 +36,15 @@ def unify (ore, p) {
 
 def unify_oredicts(ore_list) {
     for ( def ore : ore_list) {
-        if (ore) {
-            if (ore.size() > 1) {
-                unify(ore, 0)
-            }
+        if (ore && ore.size() > 1) {
+            unify(ore, 0)
```

Comment:
Undo this change, keep the two checks separate to avoid nulls causing issues

Reply:
assuming groovy isnt stupid ore should eval first and exit IF ore is null

Reply:
> Would you mind explaining what difference there is between the change and the original code for these two checks? I'm not sure I understand how there would be any difference given the order and logic of the checks are preserved in the compacted expression.

It's just a cleaner if case, should still be the same logic